packages/xcoobee-cookie-kit-react/src/CookieKitPopup.js

```diff
@@ -665,20 +679,47 @@ export default class CookieKitPopup extends React.PureComponent {
                 </div>
               </button>
               { isCountrySelectShown && (
-                <div className={`${BLOCK}__country-picker-select`}>
-                  { countryCodes.map(cCode => (
-                    <button
-                      type="button"
-                      key={`country-flag-${cCode}`}
-                      className={`xb-cookie-kit__button ${BLOCK}__country-picker-button`}
-                      onClick={() => this.handleCountryChange(cCode)}
-                      title={cCode}
-                    >
-                      <div className={`${BLOCK}__flag`}>
-                        <ReactCountryFlag code={cCode} svg />
-                      </div>
-                    </button>
-                  ))}
+                <div className={`${BLOCK}__country-picker-container`}>
+                  <button
+                    type="button"
+                    className="xb-cookie-kit__button"
+                    onClick={e => this.handleScroll(e, "up")}
+                  >
+                    <img
+                      src={caretUp}
+                      alt="caret-up"
+                    />
```

Comment:
Do you know if accessibility (aka a11y) was one of the goals of this project? If it is, then we should add some localized text for this button. This button scrolls forwards. The other button scrolls backwards. If you are not familiar with a11y, please let me know, and I will help.

Reply:
Should we add text for buttons without text? Because there are just caret icons.

Reply:
For a11y purposes, we should add some text that is only shown by screen readers. See https://getbootstrap.com/docs/3.3/getting-started/#accessibility for some explanation.

Since we are not using Bootstrap in cookie-kit, we can just add our own styles for an equivalent to `sr-only`.

I found the following in Bootstrap 3.3.7:

```
.sr-only {
    position: absolute;
    width: 1px;
    height: 1px;
    margin: -1px;
    padding: 0;
    overflow: hidden;
    clip: rect(0, 0, 0, 0);
    border: 0;
}
```

We can use these same CSS properties.

Ideally, we would want the text that is displayed to the screen reader to be localized. I am thinking it should be something like "scroll up" and "scroll down". Then whatever the equivalent is for the other 3 languages we support.

I just asked Bilal if a11y is a concern for cookie-kit. Waiting to hear back.

Reply:
It sounds like a11y isn't a primary goal for this project. Sounds like only if the changes are easy should we put effort in to make it a11y. I am going to go ahead and resolve what you have done.